### ISSUE TITLE
Geist Finance Fantom (Schema Version 2.0.1, Subgraph Version  1.03, Methodology Version 1.0.0)

### ISSUE THREAD

**Issue by dmelotik** [messari/subgraphs#862]
External QA

Creating the issue here to be tracked formally

https://docs.google.com/spreadsheets/d/1af6TL6xYhpc2sVLeuZl7dfvJ0kYE6wC7/edit?usp=sharing&ouid=103569210260619064040&rtpof=true&sd=true

**Comment by dmelotik** [comment]
Key issues:
- Revenue may need a second look in order to properly calculate (https://docs.geist.finance/#what-is-geist)
- Liquidate issues should be fixed in #834 
- Must've read the numbers wrong on accident, liquidators is 55 which is reasonable
- I am not seeing any numbers for `dailyActiveUsers`, but i see nothing wrong w the subgraph logic
- Actually deposit rewards looks accurate and borrow rewards are way under the listed APR
- rates LGTM
- Remove stables
- isActive / canBorrowFrom / canUseAsCollateral are fixed in a prev PR
- LTV / liq penalty are correct
- look into events

**Comment by dmelotik** [comment]
When comparing our total revenue to Geist's "Platform Fee" we are about $17mm short.. I am talking to their team over discord to understand how they calculate it to see if there is any discrepancy there.

**Comment by ishraq8** [comment]
Here is what i think: 
1) The protocol fees are distributed amongst GIST stakers and lockers. When it is removed before 3 months, there is a 50% penalty which is distributed amongst the remaining stakers/lockers. I think the frontend may be double counting that. I noticed that most dates their revenue tracks our subgraph. Every few days their front end has 50-100K more. This redistribution is not observed in AAVE V2, which makes me think this way.
2) Alternatively, if it is our fault. Is there any limit to how much the subgraph is reporting for revenue? 

I am further investigating that 5M revenue spike. Tryign to see which pool it is in and what events may have led to that.

**Comment by dmelotik** [comment]
@ishraq8

I think you are right here, and to confirm this I heard back from the GEIST team and they said the platform fee displayed on their website comes from... "apparently they check the events for transfers of gtokens to the staking contacts and that comes up with the platform fee numbers"

2- I mean the only limit is the data on chain from geist protocol. And the revenue is calculated the exact same way as aave v2 here. But after our discussion on this i am pretty confident this is not an issue.

**Comment by dmelotik** [comment]
90% sure i found the revenue spike issue, and it affects all aave v2 subgraphs, so I will make a separate PR and start syncing those in my hosted service

**Comment by ishraq8** [comment]
Will this effect the revenue on other aave v2 deployments. I can run a quick QA on those to double check. The data there was accurate.

**Comment by dmelotik** [comment]
in theory it should not change it by more than $10, geist CRV is an exception.

However, this change would be the proper way to do it

**Comment by dmelotik** [comment]
Going to leave the above alone! And I fixed the revenue spike (you can see here https://subgraphs.messari.io/subgraph?endpoint=https://api.thegraph.com/subgraphs/id/QmPqAh6gnNdcvC7aHwtbK8rzHyZkjhJuyQEpDKuQFqyigx&tab=protocol)

once the fix is merged it will graft to the main one and you can do a final check @ishraq8 !

**Comment by this-username-is-taken** [comment]
> in theory it should not change it by more than $10, geist CRV is an exception.
> 
> However, this change would be the proper way to do it.
> 
> cc @this-username-is-taken / @ishraq8 thoughts on leaving Aave v2 alone, and the change would affect the next aave v2 deployment. But the difference will be so small that it is not worth it to resync right now?

agreed!
